### ISSUE TITLE
Slime edges flag in 1.8.9

### ISSUE THREAD

**Issue by notKnownn** [GrimAnticheat/Grim#1389]
### Describe the false positive and how to replicate it

https://github.com/GrimAnticheat/Grim/assets/118249487/dab5b797-a043-4de0-aa6a-55439ed60e01

### Grim version

2.3.59

### Server version

I dont know, its eu.loyisa.cn (/version seems to be disabled)
Minecraft 1.8.9

### Plugins

I dont know, its eu.loyisa.cn (/version seems to be disabled)

**Comment by Souipi** [comment]
I don't have the phase false but yea antikb + simulation are falsing

**Comment by Symmettry** [comment]
isn't painsaga an altening alt lmao-

**Comment by ManInMyVan** [comment]
> isn't painsaga an altening alt lmao-

Yes

**Comment by notKnownn** [comment]
> isn't painsaga an altening alt lmao-

Yes, I use a forge mod that allows altening logging so I can test things. I will never pay mc premium
